src/python/antares_xpansion/config_loader.py

```diff
@@ -356,6 +356,10 @@ def log_level(self):
             "log_level", self._config.settings_default["log_level"]
         )
         return int(log_level_str) 
+    
+    def benders_log_file(self)-> Path:
```

Comment:
This function has the same name as `benders_log_file` in `benders_driver.py` line 170, I don't know if this is annoying